_posts/2022-05-02-the-na-bug/the-na-bug.Rmd

```diff
@@ -0,0 +1,111 @@
+---
+title: "Ooh na na... where are my sodium labs?"
+description: |
+  The NA bug, or, what happens when the same word is used in different contexts.
+author:
+  - name: Chloe Pou-Prom
+    affiliation: DSAA, Unity Health Toronto
+    affiliation_url: https://chartdatascience.ca
+date: 2022-05-02
+output:
+  distill::distill_article:
+    self_contained: false
+categories:
+  - language-R
+  - project-chartwatch
+  - post-miscellaneous
+draft: true
+preview: preview.PNG
+bibliography: references.bib
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+```{r load_date, echo = FALSE}
+library(here)
+library(xts)
+library(dygraphs)
+library(dplyr)
+library(ggplot2)
+data <- read.csv(file.path(here::here(), "_posts/2022-05-02-the-na-bug", "daily_electrolyte_counts.csv"), stringsAsFactors = FALSE) %>%
+  mutate(date = as.Date(date))
+
+daily_counts_pre_fix <- data %>%
+    filter(cohort == "pre-fix")
+
+daily_counts_post_fix <- data %>%
+    filter(cohort == "post-fix")
+```
+
+## Silent deployment
+
+Our team had been working endlessly on developing [CHARTwatch](https://www.chartdatascience.ca/the-projects/an-early-warning-system-for-general-internal-medicine-), an early warning system and in November 2019, we were ready to move to our **silent deployment** phase. This phase involved deploying our system and, as the name suggests it, running it _silently_. More concretely, our entire pipeline was running (from data extraction to data processing to model prediction), but no outputs were going to the end-user. 
```

Comment:
I used a mixture of links, citations and footnotes. This is the convention I followed:
- Website -> link
- Journal/conference article -> citation
- Comment from me -> footnote

Not sure what the best convention is to follow!

Reply:
Two recommendations for the parts: (1) endlessly sounds a bit negative, maybe something like "diligently", or something more informal like "actively"? (2) a rephrase of the following 

from "Our team had been working endlessly on developing [CHARTwatch](https://www.chartdatascience.ca/the-projects/an-early-warning-system-for-general-internal-medicine-), an early warning system and in November 2019, we were ready to move to our **silent deployment** phase."

to "Our team had been actively working on the development of [CHARTwatch](https://www.chartdatascience.ca/the-projects/an-early-warning-system-for-general-internal-medicine-), an early warning system for patients in general internal medicine at St. Michael's Hospital. In November 2019 we were ready to move to a **silent deployment** phase, which means our entire pipeline was running (from data extraction to data processing to model prediction), but no outputs were going to the end-user."